tests/jenkins/run_test_installation_docs.sh

```diff
@@ -250,6 +250,20 @@ function set_instruction_set() {
         ${sorted_indexes[$end_buildfromsource_command_index]})
 }
 
+# given a build commands string, filter any build commands that ought not be executed
+# during the test. An example would be git clone'ing the mxnet repository. You want to 
```

Comment:
nitpick: cloning* and MXNet*

Reply:
Also would like to rephrase the sentence "You want to run the tests against the current commit being tested in Jenkins" - I am unable to understand how does in this example "filter_build_command" play a role?

Reply:
I'll update the comment - the git clone'ing is to reference the git clone command. But fair comment anyway.

The issues is the build from source commands include cloning the repository, which checks out the master branch. The problem occurs when we are testing build from source commands pinned to v1.3.x against the master branch. This is why the nightly is failing.

The chosen approach to mitigate this was to filter out these git commands (and the cd into the incubator-mxnet directory). This means that the build from source commands will run directly on the version of the repository that is being tested by Jenkins and not against master.

does it make sense?

Reply:
Yup. Now it makes sense. Thanks for the clarification.